uui/components/tables/DataTable.tsx

```diff
@@ -72,21 +54,27 @@ export function DataTable<TItem, TId>(props: React.PropsWithChildren<DataTablePr
     }, [props.renderNoResultsBlock]);
 
     const onConfigurationButtonClick = React.useCallback(() => {
+        const outerProps = { columns: props.columns, columnsConfig: { ...config }, defaultConfig };
```

Comment:
rename just to 'modalProps' and pass it also in our default implementation

Reply:
we already have props and modalProps here, so I renamed it into configProps.